sectionproperties/pre/sections.py

```diff
@@ -293,224 +622,757 @@ def plot_geometry(self, ax=None, pause=True, labels=False, perimeter=False):
+        :type point_j: Tuple[float, float]
 
-        Custom section geometry.
+        :param vector: Optional. A tuple or numpy ndarray of *(x, y)* components to define the line direction.
+        :type vector: Union[Tuple[float, float], numpy.ndarray]
 
-    ..  figure:: ../images/sections/custom_mesh.png
-        :align: center
-        :scale: 75 %
+        :return: A tuple of lists containing Geometry objects that are bisected about the 
+            infinite line defined by the two given points. The first item in the tuple represents
+            the geometries on the "top" of the line (or to the "right" of the line, if vertical) and
+            the second item represents the geometries to the "bottom" of the line (or 
+            to the "left" of the line, if vertical).
 
-        Mesh generated from the above geometry.
+        :rtype: Tuple[List[Geometry], List[Geometry]]
+
+        The following example splits a 200PFC section about the y-axis::
+
+            import sectionproperties.pre.sections as sections
+            from shapely.geometry import LineString
+
+            geometry = sections.channel_section(d=200, b=75, t_f=12, t_w=6, r=12, n_r=8)
+            right_geom, left_geom = geometry.split_section((0, 0), (0, 1))
+        """
+        top_geoms_acc = []
+        bottom_geoms_acc = []
+        for geom in self.geoms:
+            top_geoms, bottom_geoms = geom.split_section(point_i, point_j, vector)
+            top_geoms_acc += top_geoms
+            bottom_geoms_acc += bottom_geoms
+        return (top_geoms_acc, bottom_geoms_acc)
+
+
+    def offset_section_perimeter(self, amount: float = 0, resolution: float = 12):
+        """Dilates or erodes perimeter of the individual geometries within the CompoundGeometry
+        object by a discrete amount. Note, because the individual geometries have their own
+        perimeters offset independently, sections don't "stick" as though they were a joined section.
+        Any aligned geometries within the CompoundGeometry will need to be re-aligned after
+        the perimeter offset.
+
+        :param amount: Distance to offset the section by. A -ve value "erodes" the section. A +ve
+            value "dilates" the section.
+        :type amount: float
+        :param resolution: Number of segments used to approximate a quarter circle around a point
+        :type resolution: float
+
+        :return: Geometry object translated to new alignment
+        :rtype: :class:`sections.pre.sections.Geometry`
+
+        The following example erodes a 200PFC section by 3::
+
+            import sectionproperties.pre.sections as sections
+
+            geometry_1 = sections.i_section(d=203, b=133, t_f=7.8, t_w=5.8, r=8.9, n_r=8)
+            geometry_2 = sections.rectangular_section(d=20, b=133)
+            compound = geometry_2.align_center(geometry_1).align_top(geometry_1) + geometry_1
```

Comment:
@connorferster, update align_top as no longer exists in code

Reply:
And that one!